src/Main.ts

```diff
@@ -1,148 +1,132 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import { BentleyError } from "@bentley/bentleyjs-core";
-import { ElectronAuthorizationBackend } from "@bentley/electron-manager/lib/ElectronBackend";
-import {
-  BriefcaseDb, BriefcaseManager, CheckpointManager, IModelHost, IModelJsFs, NativeHost, ProgressFunction, RequestNewBriefcaseArg,
-} from "@bentley/imodeljs-backend";
-import { IModelVersion, LocalBriefcaseProps, NativeAppAuthorizationConfiguration } from "@bentley/imodeljs-common";
-import { AccessToken, AuthorizedClientRequestContext } from "@bentley/itwin-client";
-
+import { BackendIModelsAccess } from "@itwin/imodels-access-backend/lib/BackendIModelsAccess";
+import { NodeCliAuthorizationClient } from "@itwin/node-cli-authorization";
 import { DataExporter } from "./DataExporter";
+import { BriefcaseManager, RequestNewBriefcaseArg } from "@itwin/core-backend/lib/cjs/BriefcaseManager";
+import { IModelDb, BriefcaseDb } from "@itwin/core-backend/lib/cjs/IModelDb";
+import { IModelHostConfiguration, IModelHost } from "@itwin/core-backend/lib/cjs/IModelHost";
+import { Logger, LogLevel } from "@itwin/core-bentley/lib/cjs/Logger";
+import { LocalBriefcaseProps, BriefcaseIdValue } from "@itwin/core-common/lib/cjs/BriefcaseTypes";
+import { IModelVersion } from "@itwin/core-common";
+import { ProgressFunction } from "@itwin/core-backend";
+import { Readline } from "readline/promises";
+
+// Find your iTwin and iModel IDs at https://developer.bentley.com/my-imodels/
+const IMODELHUB_REQUEST_PROPS = {
```

Comment:
I'm not sure why these were added ... we used to just pull the ids from the URL put in the query json.  I'd like to continue loading the ids from the json.  That way you just change the input json instead of having to rebuild.